modules/minimum-required-permissions-ipi-gcp-provided-sas.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-account.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="minimum-required-permissions-ipi-gcp-provided-sas_{context}"]
+= Required GCP permissions for user-provided service accounts
+
+When you are installing	a cluster, the compute and control plane nodes require their own service accounts.
+The default scenario occurs when the user does not provide a service account email address for the control plane or compute nodes in the `install-config.yaml` file.
+In this case the installation program will create a service account for the control plane and compute nodes, requiring additional permissions.
+For more information, see the _Installation configuration parameters for GCP_ page.
+
+[IMPORTANT]
+====
+The following roles are required for user-provided service accounts for control plane and compute nodes respectively.
+====
+
+.Required roles	for control plane nodes
+[%collapsible]
+====
+* `roles/compute.instanceAdmin`
+* `roles/compute.networkAdmin`
+* `roles/compute.securityAdmin`
+* `roles/storage.admin`
+====
+
+.Required roles for compute nodes
+[%collapsible]
+====
+* `roles/compute.viewer`
+* `roles/storage.admin`
+* `roles/artifactregistry.reader`
+====
+
+[IMPORTANT]
+====
+The following permissions are required when the user does not supply a service account for compute and control plane nodes in the `install-config.yaml` file. The permissions should be granted in the host project.
```

Comment:
Please see my [comment](https://github.com/openshift/openshift-docs/pull/81053#discussion_r1766363192) in #81053, thanks!

Reply:
I think in the interest of getting this information into the documentation for GA, we should move forward with this PR and work on reorganizing it for more clarity after GA when we have more time.

Reply:
Here is a story to track this future work: https://issues.redhat.com/browse/OSDOCS-12146